lib/install.go

```diff
@@ -347,4 +349,5 @@ func InstallProductOption(product Product, listAll, dryRun bool, customBinaryPat
 		install(product, selectVersions[selectedItx].Version, customBinaryPath, installPath, mirrorURL)
 	}
 	os.Exit(0)
+	return nil
```

Comment:
`return` after `os.Exit()` makes no sense. if this is in preparation to future changes, please leave an explanatory comment about this.

Reply:
@yermulnik has a valid point.

@MatthewJohn as long as the parent function exits appropriately, we should be good. The parent can accept the return value (nil, not nil, something else), and then decide to exit (1 or 0)

Reply:
Yes, I agree - this PR was purely the bare minimum to get the function definition update for future improvement (and even after an os.Exit, it seems go still requires a return value), so that when these new methods are released, we don't need to update the definitions, but can slowly start to remove the other means of handling errors/exiting (so we always return to the calling function with an error (or success)

Reply:
Have improved this - let me know what you think :)